pkg/controller/backup_configuration.go

```diff
@@ -255,22 +255,39 @@ func (c *StashController) EnsureBackupTriggeringCronJob(invoker apis.Invoker) er
 		// ensure that job gets deleted on completion
 		in.Spec.JobTemplate.Labels[apis.KeyDeleteJobOnCompletion] = apis.AllowDeletingJobOnCompletion
 
+		container := core.Container{
+			Name:            apis.StashContainer,
+			ImagePullPolicy: core.PullIfNotPresent,
+			Image:           image.ToContainerImage(),
+			Args: []string{
+				"create-backupsession",
+				fmt.Sprintf("--invoker-name=%s", invoker.OwnerRef.Name),
+				fmt.Sprintf("--invoker-type=%s", invoker.OwnerRef.Kind),
+			},
+		}
+		// only apply the container level runtime settings that make sense for the CronJob
+		if invoker.RuntimeSettings.Container != nil {
```

Comment:
Why not apply all the container runtime settings?

Reply:
We can't. If user specify `liveness` and `readiness` probe, then the CronJob might fail.